em-config

```diff
@@ -28,4 +28,4 @@ if [ -z "$PYTHON" ]; then
   exit 1
 fi
 
-exec "$PYTHON" "$0.py" "$@"
+exec "$PYTHON" -tt "$0.py" "$@"
```

Comment:
what does `python -tt` do? (`python --help` does not mention it)

Reply:
See the PR description

Reply:
But the PR description mentions `-t` and this is `-tt`? Is it the same?

Reply:
(and why don't I see either in python --help..? weird)

Reply:
Its in the python man page.   `-tt` is the same as `-t -t` and so the final sentence applies:  `Issue an error when the option is given twice`.

Reply:
When I run `python --help` I see `-t     : issue warnings about inconsistent tab usage (-tt: issue errors)`

Reply:
You don't see this?

Reply:
Oh.. its in the `python2 --help` output but not the `python3 --help` for some reason.

Reply:
I guess they removed it : https://github.com/python/cpython/commit/e1b5ac640841e4233bcd0204efcd50fe9b50a6d0

Reverting that part.

Reply:
Weird they removed it - very confusing...